src/modules/RA/DataLayer/components/tabs/MinisheetTab/MiniSheetFieldTree.js

```diff
@@ -88,6 +89,15 @@ const MiniSheetFieldTree = ({
     };
   }, [fields, sourceFieldId]);
 
+  const onTitleChange = useCallback(cb => ({ target: { value } }) => {
```

Comment:
The useCallback doesn't works here. Remove it or change the way it's working.

Reply:
Removed.